jobs/broken-site-report-ml/.flake8

```diff
@@ -1,2 +1,3 @@
 [flake8]
 max-line-length = 88
+ignore = E203
```

Comment:
It looks like black and flake8 disagree on the whitespace in this function `yield data[i : i + size]`
I got `E203 whitespace before ':'` error when running flake8 after black autoformat, so added this ignore rule.